### DISCUSSION TITLE
Just wanted to say I'm pretty happy with this fork

### DISCUSSION THREAD

**Discussion by hollowstrawberry** [vladmandic/sdnext#270] [General]
Glad to see an actively maintained fork with quality of life improvements. I tried using stable-diffusion-webui-ux but it's quite outdated in the backend side which lead to abysmal performance on my tight 6 GB 2060.

I love whatever you did with sdp attention, I had tried updating torch before and using sdp but experienced much higher vram usage which again made it useless on my 2060. However it works perfectly on this repo. I'm switching to it as I can disable memory attention and get deterministic results with basically no downsides.

I disagree with this repo's choice of default theme, default settings (I heard it made jpegs?) and mandatory extensions, but it's no big issue as I can change those easily and I brought over my config files.

**Comment by finefin** [comment]
hah! I was just going to post something like this :D and I totally agree!

I finally made this fork work on my machine (Lenovo notebook with RTX2060). It took some attempts and I had to manually install and set up some things - but now it's working and, whoa! this generates images fast!!! I don't really understand why I see this speed up in comparison to the A1111 version but hey, I won't complain.

I did not test everything, yet, but I got no problems so far and even my standard scripts work without any errors.

Thank you, Vladimir! Keep up the great work!

**Comment by myndxero** [comment]
I love what this guy does here, Vlad is the man. And he seems to really enjoy it whereas we probably wore Auto1111 out. Let's not wear this guy out :) Haha.

**Reply by FullBleed** [reply]
> I love what this guy does here, Vlad is the man. And he seems to really enjoy it whereas we probably wore Auto1111 out. Let's not wear this guy out :) Haha.

Not possible.  This train is moving way too fast.  ;)

But when he does start getting worn out, a little communication will go a long way.

**Comment by FullBleed** [comment]
> mandatory extensions

I wonder if it might be better to have them be "Recommended" instead... unless they are necessary for the repo to function properly.

**Reply by vladmandic** [reply]
you can still disable them in extensions tab.

**Comment by vladmandic** [comment]
> I disagree with this repo's choice of default theme, default settings (I heard it made jpegs?) and mandatory extensions, but it's no big issue as I can change those easily and I brought over my config files.

both default theme and image file format are choices. my defaults are different, but you can set whatever you want.
and extensions you can disable, just uncheck in extensions tab.
